src/aap_eda/core/tasking/__init__.py

```diff
@@ -172,14 +173,16 @@ def __init__(
         if queue_class is None:
             queue_class = Queue
 
+        queue_name = settings.RULEBOOK_QUEUE_NAME
```

Comment:
The queue name is not unique?  Perhaps only until future changes, but then a comment to that affect would be good.

Reply:
I'm not sure if I'm following you. RULEBOOK_QUEUE_NAME is set in the worker node. Each node should have a different name. Uniqueness is handled at the deployment time (we will use /etc/machine-id)